client/my-sites/google-my-business/google-my-business-stats/index.js

```diff
@@ -0,0 +1,325 @@
+						<div className="google-my-business-stats__not-enough-data">
+							<img src="/calypso/images/google-my-business/not-enough-data.svg" />
+							<br />
+							{ translate( "We don't have enough data" ) }
+						</div>
+					</Card>
+
+					<Card>
+						<div className="google-my-business-stats__feature">
+							<p>
+								Customers compare business listings on Google to make decisions. Make your listing
+								count.
+							</p>
+							<img
+								src="/calypso/images/google-my-business/compare.svg"
+								className="google-my-business-stats__feature-image"
+							/>
+						</div>
+						<Button primary className="google-my-business-stats__feature-button">
+							Complete Your Listing
+							<Gridicon icon="external" />
+						</Button>
+					</Card>
+
+					<SectionHeader label={ translate( 'Return customers' ) } />
+					<Card>
+						<p className="google-my-business-stats__description">
+							How many customers return to your website?
+						</p>
+						<FormFieldset>
+							<FormSelect className="google-my-business-stats__select">
+								<option>{ translate( '1 week' ) }</option>
+								<option>{ translate( '1 month' ) }</option>
+								<option>{ translate( '1 quarter' ) }</option>
+							</FormSelect>
+						</FormFieldset>
+						<div className="google-my-business-stats__not-enough-data">
+							<img src="/calypso/images/google-my-business/not-enough-data.svg" />
+							<br />
+							{ translate( "We don't have enough data" ) }
+						</div>
+					</Card>
+
+					<SectionHeader label={ translate( 'Popular times' ) } />
+					<Card>
+						<p className="google-my-business-stats__description">
+							The most popular times for your customers to visit your website.
+						</p>
+						<FormFieldset>
+							<FormSelect className="google-my-business-stats__select">
+								<option>{ translate( '1 week' ) }</option>
+								<option>{ translate( '1 month' ) }</option>
+								<option>{ translate( '1 quarter' ) }</option>
+							</FormSelect>
+						</FormFieldset>
+						<div className="google-my-business-stats__not-enough-data">
+							<img src="/calypso/images/google-my-business/not-enough-data.svg" />
+							<br />
+							{ translate( "We don't have enough data" ) }
+						</div>
+					</Card>
+
+					<SectionHeader label={ translate( 'Photo views' ) } />
+					<Card>
+						<p className="google-my-business-stats__description">
+							The number of times your business photos have been viewed, compared to photos from
+							other businesses.
+						</p>
+						<FormFieldset>
+							<FormSelect className="google-my-business-stats__select">
+								<option>{ translate( '1 week' ) }</option>
+								<option>{ translate( '1 month' ) }</option>
+								<option>{ translate( '1 quarter' ) }</option>
+							</FormSelect>
+						</FormFieldset>
+						<img src="/calypso/images/google-my-business/bar-chart.svg" />
+					</Card>
+
+					<SectionHeader label={ translate( 'Photo quantity' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **12** times:
`translate( 'Quantity:' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gmb-prototype)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).